src/cow-react/common/hooks/useAllTokensList.ts

```diff
@@ -0,0 +1,9 @@
+import { Token } from '@uniswap/sdk-core'
+import { useMemo } from 'react'
+import { useAllTokens } from 'hooks/Tokens'
+
+export function useAllTokensList(): Token[] {
+  const allTokens = useAllTokens()
+
+  return useMemo(() => Object.values(allTokens), [allTokens])
```

Comment:
Would the json stringify for the deps tactic help here?

Reply:
It's not necessary, because we have it inside of `useAllTokens`